src/examples/src/widgets/speed-dial/Position.tsx

```diff
@@ -11,18 +11,11 @@ type Directions = 'right' | 'left' | 'up' | 'down';
 export default factory(function Direction({ middleware: { icache } }) {
 	const position: SpeedDialPositions = icache.getOrSet('position', 'bottom-right');
 	const supportedDirections = getSupportedDirections(position);
-	let direction: Directions | undefined = icache.get('direction');
+	let direction: Directions | undefined = icache.getOrSet('direction', 'left');
```

Comment:
Since the most common use case for the `SpeedDial` is at the bottom right of the screen, the default `position` and `direction` were updated to reflect this (along with applicable examples, as seen here.)